client/post-editor/media-modal/image-editor/image-editor-canvas.jsx

```diff
@@ -68,7 +68,9 @@ const MediaModalImageEditorCanvas = React.createClass( {
 	},
 
 	componentWillReceiveProps( newProps ) {
-		this.getImage( newProps.src );
```

Comment:
This was causing the image editor (+ media modal) slowness on any action as image had to be loaded and processed on any re-render. For some reason, I thought that [this lifecycle method](https://facebook.github.io/react/docs/component-specs.html#updating-componentwillreceiveprops) is called only on receiving **new** props. My bad..